src/app/children/notes/notes.component.ts

```diff
@@ -49,8 +49,9 @@ export class NotesComponent implements OnInit {
 
 
   generateNewRecordFactory() {
-    // define values locally because "this" is a different scope after passing a function as input to another component
-    const user = this.sessionService.getCurrentUser().name;
+    // define values locally because "this" is a different scope after passing a function as input to another component#
+    // TODO: currentUser undefined in tests
```

Comment:
if it's just required for tests, we should try to mock the currentUser value.
But I think your change makes the code more stable in general and is a good addition :+1:

Reply:
Agreed, but I probably won't fix this on this branch, since its tangential to the blob service. (I will open a new issue for this.)

Reply:
as I said, your change is good in general - so I think we can also just remove the TODO here

Reply:
Ok :) Will do that.